tensortrade/exchanges/simulated/simulated_exchange.py

```diff
@@ -33,23 +33,39 @@ class SimulatedExchange(Exchange):
     """
 
     def __init__(self, data_frame: pd.DataFrame = None, **kwargs):
-        super().__init__(
-            dtype=self.default('dtype', np.float32),
-            feature_pipeline=self.default('feature_pipeline', None),
-            **kwargs
-        )
-
-        self._commission_percent = self.default('commission_percent', 0.3, kwargs)
-        self._base_precision = self.default('base_precision', 2, kwargs)
-        self._instrument_precision = self.default('instrument_precision', 8, kwargs)
-        self._initial_balance = self.default('initial_balance', 1e4, kwargs)
-        self._price_column = self.default('price_column', 'close', kwargs)
-        self._pretransform = self.default('pretransform', True, kwargs)
-
-        self.data_frame = self.default('data_frame', data_frame)
-
-        model = self.default('slippage_model', 'uniform', kwargs)
-        self._slippage_model = slippage.get(model) if isinstance(model, str) else model()
+        super().__init__(base_instrument=kwargs.get('base_instrument', 'USD'),
+                         dtype=kwargs.get('dtype', np.float16),
+                         feature_pipeline=kwargs.get('feature_pipeline', None))
+
+        self._commission_percent = kwargs.get('commission_percent', 0.3)
+        self._base_precision = kwargs.get('base_precision', 2)
+        self._instrument_precision = kwargs.get('instrument_precision', 8)
+        self._min_trade_price = kwargs.get('min_trade_price', 1E-6)
```

Comment:
You will want to update this branch to master, because these `_min` and `_max` values are set in `Exchange` currently.